test/orchard/inspect_test.clj

```diff
@@ -205,13 +222,13 @@
 
 (deftest path-test
   (testing "inspector tracks the path in the data structure"
-    (is (-> long-map inspect (inspect/down 39) render ^String (first) (.endsWith "\"  Path: (find 50) key\")")))
-    (is (-> long-map inspect (inspect/down 40) render ^String (first) (.endsWith "\"  Path: (get 50)\")")))
-    (is (-> long-map inspect (inspect/down 40) (inspect/down 0) render ^String (first) (.endsWith "\"  Path: (get 50) class\")"))))
+    (is (= "  Path: (find 50) key" (-> long-map inspect (inspect/down 39) render last)))
+    (is (= "  Path: (get 50)" (-> long-map inspect (inspect/down 40) render last)))
+    (is (= "  Path: (get 50) class"  (-> long-map inspect (inspect/down 40) (inspect/down 0) render last))))
   (testing "doesn't show path if unknown navigation has happened"
-    (is (-> long-map inspect (inspect/down 40) (inspect/down 0) (inspect/down 1) render ^String (first) (.endsWith "(:newline))"))))
+    (is (= '(:newline)  (-> long-map inspect (inspect/down 40) (inspect/down 0) (inspect/down 1) render last))))
```

Comment:
I like these new assertions, they're simpler.

Just in case, why is `first` now `last`?

Reply:
Previously render returned a vector with a single string. So what the test did, it used first to get at the string, and then matches that large string with endsWith. Since we now return a seq, extracting the first element is not needed (we are interested in the last one). The test diff looks kind of confusing